data/policies/full_system/README.md

```diff
@@ -0,0 +1,27 @@
+This directory contains a sample full system cascade policy intended for
+demonstrating and testing Cascade.
+
+The intent is to create a policy that boots in enforcing mode and allows simple
```

Comment:
We should also say which distro & version this was tested against.  Maybe any big configuration changes that were made.

Reply:
Added a line, and also fleshed out the README.md quite a bit.